README.md

```diff
@@ -94,10 +94,22 @@ all cases, but for the 2NT bids we use Puppet Stayman.
 * 2♦️ Constructive raise
 * 3♦️ Preemptive raise
 * If responder is a passed hand, all responses are nonforcing.
+* After a 1♦️-1M-1NT sequence showing 11-13 balanced, 2 way checkback is on
```

Comment:
Add a link to "2 way checkback". Link to video is good: https://www.youtube.com/watch?v=5_RD8ycgRT8

Reply:
Should be done now.